src/code/z_eff_blure.c

```diff
@@ -421,7 +422,9 @@ void EffectBlure_DrawElemNoInterpolation(EffectBlure* this, EffectBlureElement*
         vtx[0].v.cn[2] = sp78.b;
         vtx[0].v.cn[3] = sp78.a;
 
-        if (1) {} // Necessary to match
+#if OOT_VERSION == OOT_GC_EU_MQ_DBG
+        if (1) {}
+#endif
```

Comment:
The "debug-only" empty if can be removed with
```c
    if (vtx == NULL) {
        // "Vertices cannot be secured."
        PRINTF("z_eff_blure.c::SQ_NoInterpolate_disp() 頂点確保できず。\n");
        goto end;
    }
```

I gave up trying to remove the second one, but it seems now it doesn't have to be N64-only at least. Also I'd move it down a bit, just before the gSPVertex call

Reply:
that is an interesting pattern, the goto end;
it's also probably real
I would like to clean this up in another PR (or you can) as it involves significant un-indent changes and isn't related to ntsc-1.2 directly

Reply:
I do think it's related to ntsc-1.2, a large part of it is finding better matches for these draw functions. What's the difference between unindent now vs later?

Reply:
Also CI is failing probably because of the `OOT_` prefix on version names (but this should use probably use OOT_DEBUG anyway)

Reply:
> I do think it's related to ntsc-1.2, a large part of it is finding better matches for these draw functions. What's the difference between unindent now vs later?

there's none really, but I'd like to look into using this pattern for other functions too, which creeps out of scope

Reply:
I'll settle for a TODO though. It should be `#if OOT_DEBUG` though, since this difference is probably because of -g3 and not something specific to gc-eu-mq-dbg

Reply:
Thanks :)